python-avd/pyavd/_eos_designs/structured_config/base/platform_mixin.py

```diff
@@ -0,0 +1,133 @@
+        # Also iterate through member interfaces of all L3 Port-Channels with 'rx_queue' config
+        if not self.shared_utils.is_sfe_interface_profile_supported:
+            return None
+        sfe_profile_member_interfaces = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.Interfaces()
+        # iterate through each L3 interface
+        for interface in self.shared_utils.l3_interfaces:
+            intf_profile_settings = self._build_interface_with_rx_queue_settings(interface)
+            if intf_profile_settings is not None:
+                sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        # iterate through member interfaces of each L3 Port-Channel
+        for interface in self.shared_utils.node_config.l3_port_channels:
+            for member_intf in interface.member_interfaces:
+                intf_profile_settings = self._build_interface_with_rx_queue_settings(member_intf)
+                if intf_profile_settings is not None:
+                    sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        if sfe_profile_member_interfaces:
+            return sfe_profile_member_interfaces._natural_sorted()
+        return None
+
+    def _build_interface_with_rx_queue_settings(
+        self: AvdStructuredConfigBaseProtocol,
+        l3_member_interface: EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3InterfacesItem
+        | EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3PortChannelsItem.MemberInterfacesItem,
+    ) -> EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem | None:
+        """Returns one SFE interface with profile settings for structured config."""
+        if not l3_member_interface.rx_queue:
+            # specified interface does not have explicit "rx_queue" key or sub-keys specified
+            # exclude such interface from profile being built
+            return None
+        # validate rx_queue 'count' when specified
+        self._validate_rx_queue_count(l3_member_interface.rx_queue.count, l3_member_interface.name)
+
+        rx_queue_workers = set()
+        for _worker in l3_member_interface.rx_queue.workers:
+            for worker_id in map(int, range_expand(_worker)):
+                self._validate_rx_queue_worker(worker_id, l3_member_interface.name)
+                rx_queue_workers.add(worker_id)
+        rx_queue_workers_range = list_compress(list(rx_queue_workers))
+
+        intf_profile_settings = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem()
+        intf_profile_settings.name = l3_member_interface.name
+        intf_profile_settings.rx_queue.count = l3_member_interface.rx_queue.count
+        if rx_queue_workers_range:
+            intf_profile_settings.rx_queue.worker = rx_queue_workers_range
+        intf_profile_settings.rx_queue.mode = l3_member_interface.rx_queue.mode
+        return intf_profile_settings
```

Comment:
Can it not be created like this?
```suggestion
        return EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem(
            name = l3_member_interface.name.
            rx_queue.count = l3_member_interface.rx_queue.count
            rx_queue = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem.RxQueue(
                worker = rx_queue_workers_range
                mode = l3_member_interface.rx_queue.mode
            )
        )
```